app/cafe.py

```diff
@@ -0,0 +1,21 @@
+import datetime
+
+from app.errors import (
+    NotWearingMaskError,
+    OutdatedVaccineError,
+    NotVaccinatedError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str | Exception:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("Visitor wasn't vaccinated")
+        if visitor["vaccine"]["expiration_date"] < datetime.date.today():
```

Comment:
Use `get` to avoid `KeyError`
```suggestion
        if visitor["vaccine"].get("expiration_date") < datetime.date.today():
```